x-pack/packages/kbn-elastic-assistant/impl/assistant/helpers.ts

```diff
@@ -63,36 +78,13 @@ export const getBlockBotConversation = (
  * @param connectors
  */
 export const getDefaultConnector = (
-  connectors: Array<ActionConnector<Record<string, unknown>, Record<string, unknown>>> | undefined
-): ActionConnector<Record<string, unknown>, Record<string, unknown>> | undefined =>
-  connectors?.length === 1 ? connectors[0] : undefined;
-
-/**
- * When `content` is a JSON string, prefixed with "```json\n"
- * and suffixed with "\n```", this function will attempt to parse it and return
- * the `action_input` property if it exists.
- */
-export const getFormattedMessageContent = (content: string): string => {
```

Comment:
I'm wondering if we should have not deleted this? @andrew-goldstein 

I see an error in our telemetry logs like:

```
Failed to parse. Text: "```json { "action": "Final Answer", "action_input": "Here are a few more example ES|QL queries for different data exfiltration scenarios:\n\n1. Detect large volumes of network traffic by user and process:\n```\nFROM logs-*\n| WHERE event.category == \"network\" AND agent.type == \"windows\"\n| STATS bytes = sum(network.bytes) BY process.name, user.name, source.ip, destination.ip\n| EVAL mb = bytes / (1024 * 1024)\n| WHERE mb > <network_threshold_value>\n| SORT mb DESC\n| KEEP process.name, user.name, source.ip, destination.ip, mb\n```\n2. Detect unusual file creation activity:\n```\nFROM logs-*\n| WHERE event.category == \"file\" AND event.action == \"create\" AND agent.type == \"windows\"\n| STATS count() BY process.name, user.name, host.name, file.extension\n| WHERE count() > <file_creation_threshold_value>\n| SORT count() DESC\n| KEEP process.name, user.name, host.name, file.extension, count()\n```\n3. Detect suspicious email attachments:\n```\nFROM logs-*\n| WHERE event.category == \"email\" AND event.action == \"send\" AND agent.type == \"windows\"\n| WHERE file.extension IN (\\".exe\", \\".bat\", \\".com\", \\".dll\", \\".docm\", \\".jar\", \\".js\", \\".jse\", \\".lnk\", \\".msi\", \\".ps\", \\".ps1\", \\".scr\", \\".vbs\", \\".wsf\")\n| STATS count() BY process.name, user.name, source.email, destination.email, file.name\n| WHERE count() > <email_attachment_threshold_value>\n| SORT count() DESC\n| KEEP process.name, user.name, source.email, destination.email, file.name, count()\n```\nRemember to replace `<network_threshold_value>`, `<file_creation_threshold_value>`, and `<email_attachment_threshold_value>` with values appropriate for your environment. Adjust the queries as needed based on your specific logging setup and fields." } ```". Error: SyntaxError: Expected ',' or '}' after property value in JSON at position 1101
```

Could this be coming from us?

Reply:
@stephmilovic , that was from my cluster.

Reply:
but it is an 8.13 cluster, before this code was deleted.